bridge/client/app/_views/ktb-dashboard-legacy-view/ktb-dashboard-legacy-view.component.scss

```
@@ -1,5 +1,5 @@
-@import '../../variables';
-@import '~@dynatrace/barista-components/core/src/style/variables';
+@import 'client/variables';
+@import 'node_modules/@dynatrace/barista-components/core/src/style/variables';
```

Could you change it to be relative again?
```suggestion
@import '../../../variables';
@import '~@dynatrace/barista-components/core/src/style/variables';
```